### ISSUE TITLE
Fails with comments in multiline declaration

### ISSUE THREAD

**Issue by fbergroth** [Nykakin/chompjs#21]
This fails to parse:
```
var obj = {
  // Comment
  x: "X", // Comment
};
```

**Comment by Nykakin** [comment]
Fixed by #22, should be included in next release
```python
>>> data = """
... var obj = {
...   // Comment
...   x: "X", // Comment
... };
... """
>>> chompjs.parse_js_object(data)
{'x': 'X'}
```

**Comment by fbergroth** [comment]
Great! Can you release wheels?

**Comment by Nykakin** [comment]
Released as 1.1.2 version. Please let me know if you encounter any problems.
